internal/model/sql/postgres/schema/hellper.sql

```diff
@@ -19,6 +19,7 @@ CREATE TABLE public.incident (
 	severity_level int4 NULL,
 	channel_name text NULL,
 	updated_at timestamp NOT NULL DEFAULT now(),
+  snoozed_at timestamptz NULL,
```

Comment:
is it correct? or would it be "timestamp"
for this field is it important to evaluate the timezone ?

Reply:
[This commit](https://github.com/ResultadosDigitais/hellper/pull/84/commits/ba65079a7e164f040c479295e6982e611d8f5333) change to **timestamp**

Reply:
@tiagoemsi I don't know the best option here too, can you explain why use `timestamptz`?

Reply:
The `timestamptz` data is the timestamp with time zone. The PostgreSQL provides two temporal data types for handling timestamp, one **without** timezone (`timestamp`) and one **with** timezone (`timestamptz`).

```sql
SET timezone = 'America/Los_Angeles';

SELECT ts, tstz FROM	timestampz_demo;

         ts          |          tstz
---------------------+------------------------
 2016-06-22 19:10:25 | 2016-06-22 19:10:25-07
(1 row)
```

I recommend using the same pattern:

```sql
	[...]
	start_ts timestamptz NULL,
	end_ts timestamptz NULL,
	identification_ts timestamptz NULL,
	[...]
```

Reply:
We can use `timestamp`, no problem, but I also recommend changing the others

Reply:
OK, I agreed in using either, but the same for all.